test/verify/check-pages

```diff
@@ -862,6 +865,84 @@ OnCalendar=daily
         # logging out too fast, some D-Bus services get disconnected
         self.allow_restart_journal_messages()
 
+    @testlib.skipBrowser(".upload_files() doesn't work on Firefox", "firefox")
+    def testUpload(self) -> None:
```

Comment:
Please skip in test/browser/run-test.sh, this doesn't test external APIs.

Reply:
I think I need help understanding this.

Reply:
Nevermind, sorry! It already has
```
    # Don't run TestPages, TestPackages, and TestTerminal at all -- not testing external APIs
```
I.e. we don't run check-pages there at all.